spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -782,7 +782,7 @@ bom {
 		links {
 			site(version -> "https://jakarta.ee/specifications/servlet/%s.%s"
 				.formatted(version.major(), version.minor()))
-			javadoc(version -> "https://jakarta.ee/specifications/servlet/%s.%s/apidocs"
+			javadoc(version -> "https://jakarta.ee/specifications/servlet/%s.%s/apidocs/jakarta.servlet"
```

Comment:
Instead of appending "jakarta.persistence" to the url, here you could leverage the third argument to .formatted(...) and use:

`"https://jakarta.ee/specifications/servlet/%s.%s/apidocs/%s".formatted(...)`